claimer.js

```diff
@@ -133,23 +140,34 @@ function sleep(delay) {
                     offer.date = Date.now();
                     claimedPromos.push(offer);
                 } catch (err) {
+                    notificationMessages.push(`${email} failed to claim ${offer.title}`);
                     Logger.warn(`Failed to claim ${offer.title} (${err})`);
                     if (err.response
                         && err.response.body
                         && err.response.body.errorCode === "errors.com.epicgames.purchase.purchase.captcha.challenge") {
                         // It's pointless to try next one as we'll be asked for captcha again.
-                        Logger.error("Aborting!");
+                        let errMess = "Aborting! Captcha detected.";
+                        notificationMessages.push(errMess);
+                        Logger.error(errMess);
                         break;
                     }
                 }
             }
 
             History[email] = claimedPromos;
-            notify(appriseUrl, newlyClaimedPromos);
+
+            // Setting up notification message for current account
+            if (newlyClaimedPromos.length > 0) {
+                notificationMessages.push(`${email} claimed ${newlyClaimedPromos.length} freebies: ${
+                    newlyClaimedPromos.join(", ")}`);
+            } else {
+                notificationMessages.push(`${email} has no unclaimed freebies`);
```

Comment:
This may not entirely be accurate. It could be there are unclaimed freebies, but it failed to claim it.

Reply:
I see. I have logged the failed claim in line 143, so maybe this line will be changed to: `notificationMessages.push(`${email} has claimed 0 freebies`);`

What do you think?

Reply:
Yeah, that should do it.